### PR TITLE
[ticket/16133] Update npm packages to latest versions

### PR DESCRIPTION
**Pull request by marc1706**
PHPBB3-16133

Checklist:

- [x] Correct branch: master for new features; 3.3.x & 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html), [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-16133

### CODE REVIEW

**Comment by marc1706** [comment]
@hanakin apparently gulp-cssnano is deprecated and recommends using cssnano via postcss: https://github.com/ben-eb/gulp-cssnano

**Comment by marc1706** [comment]
This might help: https://github.com/yakamara/gulp-tasks/commit/46dddad5fea76e4edd5b02a476ded4915672c9b5

**Comment by marc1706** [comment]
I'm also not able to run gulp after these updates. Not sure if it was broken before or was broken as a result of the updates though (will have to check).

**Comment by hanakin** [comment]
@marc I will take a look at it. Im just getting back into things after my vacation. Th emore we can run through postcss the better in my opinion. I also want to look in to not having to use gulp at all in favor of something more stable. possibly elixir or something else. As for the gulp not running. It happens when changing versions sometimes.

File: package.json [hunk 1]
```diff
@@ -46,22 +46,23 @@
...
-		"braces": "^2.3.1",
-		"del": "^3.0.0",
-		"gulp": "^4.0.0",
-		"gulp-autoprefixer": "^4.0.0",
-		"gulp-cssnano": "^2.1.2",
-		"gulp-postcss": "^7.0.0",
-		"gulp-rename": "^1.2.2",
-		"gulp-sass": "^3.1.0",
-		"gulp-sourcemaps": "^2.6.1",
+		"acorn": "^7.0.0",
```
Comment (hanakin) [inline package.json]: can be removed

File: package.json [hunk 2]
```diff
@@ -46,22 +46,23 @@
...
+		"braces": "^3.0.2",
```
Comment (hanakin) [inline package.json]: can be removed

File: package.json [hunk 3]
```diff
@@ -46,22 +46,23 @@
...
+		"cssnano": "^4.1.10",
+		"del": "^5.0.0",
+		"gulp": "^4.0.2",
+		"gulp-autoprefixer": "^7.0.0",
+		"gulp-postcss": "^8.0.0",
+		"gulp-rename": "^1.4.0",
+		"gulp-sass": "^4.0.2",
+		"gulp-sourcemaps": "^2.6.5",
 		"gulp-stylefmt": "^1.1.0",
```
Comment (hanakin) [inline package.json]: remove this

File: gulpfile.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ const autoprefixer = require('gulp-autoprefixer');
 const sass = require('gulp-sass');
 const rename = require('gulp-rename');
 const sourcemaps = require('gulp-sourcemaps');
-const cssnano = require('gulp-cssnano');
+const cssnano = require('cssnano');
 const postcss = require('gulp-postcss');
 const stylefmt = require('gulp-stylefmt');
```
Comment (hanakin) [inline gulpfile.js]: remove this

Comment (hanakin) [reply gulpfile.js]: and the call on line 37

**Pull request merged**